test/liveValidation/swaggers/specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-01-01/apimapis.json

```diff
@@ -4698,6 +4703,7 @@
     },
     "ApiIdRevParameter": {
       "name": "apiId",
+      "readOnly": true,
```

Comment:
can we avoid modifying real swagger? especially this path parameter

Reply:
it is not changed in the latest commit